src/SelfService/Domain/Models/Capability.cs

```diff
@@ -8,17 +8,17 @@ public class Capability : AggregateRoot<CapabilityId>
         CapabilityId id,
         string name,
         string description,
-        DateTime? deleted,
         DateTime createdAt,
         string createdBy
     )
         : base(id)
     {
         Name = name;
         Description = description;
-        Deleted = deleted;
         CreatedAt = createdAt;
+        Status = CapabilityStatusOptions.Active; // all new capabilities are active to begin with
         CreatedBy = createdBy;
+        ModifiedAt = createdAt; // this will always be the same as CreatedAt for a new Capability
```

Comment:
In continuation of our discussion of how and when to add comments. In this scenario I think you should omit them, for me it is obvious from the code, but up to you :)